src/components/searchbox/SearchBox.jsx

```diff
@@ -113,8 +117,14 @@ function CustomSearchBox(props) {
           onChange={(event) => {
             refineFunction(event.currentTarget.value);
             refine(event.currentTarget.value);
+            setTooltip(false);
           }}
         />
+        {!!tooltip && (
+          <div className="searchbox__tooltip">
+            <p>{t('tooltip')}</p>
+          </div>
+        )}
         {!!hasKeystroke && (
```

Comment:
same here, isnt this already a boolean?

Reply:
Just tooltip can return a 0 when component is mounted. with !! we're forcing to return a boolean 😊